crpg.py

```diff
@@ -99,7 +101,7 @@ def generate(filename):
         for node in nodes.split("\n"):
             n, n_type, *args = re.split(r"\s*\|\s*", node)
 
-            if n_type == "fight" or n_type == "run":
+            if n_type == "fight" or n_type == "run" or n_type[:2] == "po" or n_type == "currency":
```

Comment:
If this keeps expanding, we may want to expand this into a map, or (like you did) add a prefix to each type!

Reply:
Sure!